trace/ptrace/ptrace.go

```diff
@@ -873,6 +944,9 @@ func processEvents(r *exptrace.Reader, tr *Trace, progress func(float64)) error
 		}
 	}
 
+	tr.Metrics["/gotraceui/goroutine/total:goroutine"] = gm.totalGoroutines.finalize()
+	tr.Metrics["/gotraceui/goroutine/runnable:goroutine"] = gm.runnableGoroutines.finalize()
```

Comment:
We probably want the other possible states, too. So, instead of `total` and `runnable`, we probably want `running`, `runnable`, and `blocked`, the sum of which implicitly is the total.

As for the names of the metrics, let's use the ones from https://github.com/golang/go/issues/15490#issuecomment-1636156744 (but with the `/gotraceui/` prefix)